app/views/lifetimeallowance/LtaProtectionOrEnhancementsView.scala.html

```diff
@@ -0,0 +1,60 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukRadios: GovukRadios,
+    govukButton: GovukButton
+)
+
+@(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
+@import controllers.lifetimeallowance.routes
+
+@protectionUrl = {
+"https://www.tax.service.gov.uk/gg/sign-in?continue=/protect-your-lifetime-allowance/existing-protections&origin=pensions-lifetime-allowance-frontend"
+}
+
+@contentHint = {
+        <p class = "govuk-body">@messages("ltaProtectionOrEnhancements.hint1")</p>
+        <p><a target="_blank" rel="noopener" href=@protectionUrl> @messages("ltaProtectionOrEnhancements.hint2") </a></p>
```

Comment:
Add this class for the vertical line class="govuk-inset-text"

Reply:
Done